installers/linux/archlinux/PKGBUILD

```diff
@@ -1,8 +1,11 @@
 # vim: ts=2 sts=2 sw=2 et ft=sh
```

Comment:
They get auto-generated by the jenkins script, which should add the changes in the release PR it creates.   Theres an additional step of taking these files and moving them to the arch linux repo and pushing.

Maybe we could just do that instead of this intermediate staging area in the minikube repo